Lib9c.Models.Tests/Items/EquipmentTest.cs

```diff
@@ -6,7 +6,7 @@ namespace Lib9c.Models.Tests.Items;
 
 public class EquipmentTest
 {
-    [Fact]
+    [Fact(Skip = "Failed")]
```

Comment:
Since xunit doesn't provide API reference website, there is source code link

https://github.com/xunit/xunit/blob/685423ea7942d71ef4f6781ff3fc216c0257e64c/src/xunit.v3.core/Abstractions/Attributes/IFactAttribute.cs#L28-L36